ch05/couples.ml

```diff
@@ -35,8 +35,8 @@ let rec rem_anchovy = function
 
 let rec rem_tuna = function
   | Bottom            -> Bottom
-  | Topping (Tuna, x) -> rem_anchovy x
-  | Topping (f, x)    -> Topping (f, rem_anchovy x)
+  | Topping (Tuna, x) -> rem_tuna x
+  | Topping (f, x)    -> Topping (f, rem_tuna x)
```

Comment:
clearly i intended to do the right thing, since i `let rec` ... i just got sloppy